app/src/main/java/com/github/orelzion/spotifyclone/ClassesExtentions.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.orelzion.spotifyclone
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.commit
+
+class ClassesExtentions {
```

Comment:
great! 
You can actually declare this file as `object` instead of `class`, and then you don't need to add a companion object